home.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home Page</title>
+    <link rel="stylesheet" href="./assets/styles/homeStyle.css">
+</head>
+<body>
+    <head>
+        <div class="main-container">
+        <div class="mobile-nav">
+            <div class="moblile-menu-icon">
+                <!-- <a id="mobile-icon-btn" href="#"> <img src="./assets/images/mobile_menu.png" alt="menu hamburger icon"></a> -->
+            </div>
+            <div id="myNav" class="overlay">
+                <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+                <div class="overlay-content">
+                    <a href="./About.html" onclick="closeNav()">About</a>
+                    <a href="#about-social" onclick="closeNav()">Program</a>
+                    <a href="#contact" onclick="closeNav()">Join</a>
+                    <a href="#News" onclick="closeNav()">News</a>
+                    <a href="#contact" onclick="closeNav()">CC Campaign</a>
+                </div>
+            </div>
+            <div class="desk-nav">
+                <div class="logo-png">
+                    <a href="#"> <img src="./assets/images/logopng.png" alt="logo image"> </a>
+                </div>
+                <div class="desk-nav-items">
+                    <ul>
+                        <li class="menu-item"><a href="./About.html"> About </a> </li>
+                        <li class="menu-item">Progam</li>
+                        <li class="menu-item">Join</li>
+                        <li class="menu-item">Sponsor</li>
+                        <li class="menu-item">News</li>
+                    </ul>
+                </div>
+            </div>
+            <nav class="flex-nav"><button onclick="openNav()">☰</button></nav>
+        </div>
+    </head>
+    <!-- coding the hero section here -->
+        <section class="hero-section">
+            <img id="main-img" src="./assets/images/main-image.png" alt="">
+            <div class="message-main">
+                <p>
+                    Experts from all around the globe, gathering to share their experiences, and to invest in your ideas, your dream is coming true, just prepare your pitch and impress one investor at least to get the fund
+                </p>
+            </div>
+            <div class="location-data">
+                <p id="date">2022.10.14 (Wed) ~ 17 (Sat)</p>
+                <p id="place">@ National Museum of Korea, Art Center Nabi, etc.</p>
+            </div>
```

Comment:
- [ ] Kindly ensure that your hero section in the mobile view is centralized on the viewport and also make sure the content of the hero section matches the design guidelines given in terms of margins( i.e The section has too many margins), fonts (i.e Your fonts are too small) and the font-weight so that your website can look better 😄 

| Design Guideline  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/190512153-9aa9a7cd-9df8-4b8b-b0f8-529608ca2165.png)  | ![image](https://user-images.githubusercontent.com/63473584/190512294-eb621867-4921-4f82-94dd-abe3f908b598.png)  |